pkg/client/core/report/console/application.go

```diff
@@ -0,0 +1,43 @@
+package console
+
+import (
+	"io"
+
+	"github.com/logrusorgru/aurora/v3"
+	"github.com/oslokommune/okctl/pkg/client"
+	"github.com/oslokommune/okctl/pkg/client/store"
+	"github.com/oslokommune/okctl/pkg/spinner"
+)
+
+type applicationReport struct {
+	console *Console
+}
+
+// ReportDeleteApplication produces relevant output upon application deletion
+func (r *applicationReport) ReportDeleteApplication(reports []*store.Report) error {
```

Comment:
Go question which I maybe should post somewhere else: Since ReportDeleteApplication is a method defined on the interface client.ApplicationReport, shouldn't the documentation be placed on the interface instead of this specific method? The comment for this function will be readonly only by consumers in the same package.

Perhaps both could be documented if there was a slight difference between an applicationReport and a client.ApplicationReport.

Reply:
Exported functions are documented in the implementation, it seems this is how the linter also expects things to be, and I assume its part of how go generats docs ...

Reply:
If this is intended, it's fine. However, it seems strange to me to document the implementation (which only the package developers would read) and not the exported function, as people using your package are the ones I would imagine actually need documentation.

For instance, when using calling this method in [service_application.go](https://github.com/oslokommune/okctl/pull/116/files#diff-377a344d9c4a37a63ab3667b24a73fe7db6e97f12c8d100b76eeb0dd2aab1e56R36), you don't get any documentation.

Reply:
Makes sense @yngvark. I didn't really think about it. I'll move the comments